### ISSUE TITLE
About loss

### ISSUE THREAD

**Issue by cancanzhu** [Shuijing725/CrowdNav_Prediction_AttnGraph#20]
Hello, I would like to ask why the loss fluctuates so dramatically. Does this have any impact on the training? Is the model converging?

**Comment by Shuijing725** [comment]
If the value loss is large and fluctuating, it's fine, as long as the reward converges to 20~25.

**Comment by cancanzhu** [comment]
Thank you very much for your response! 
Your work is outstanding.  I have just started learning about reinforcement learning navigation and I was really impressed by your paper. 
I have another question I would like to ask: 
How were the qualitative evaluation metrics, such as success rate, for the comparative experiments in the paper calculated? Could you provide me with some suggestions for modifications?

**Comment by Shuijing725** [comment]
The testing metrics are in [evaluation.py](https://github.com/Shuijing725/CrowdNav_Prediction_AttnGraph/blob/main/rl/evaluation.py)

**Comment by cancanzhu** [comment]
Thank you! Does the spatial_edges variable in the selfAttn_srnn_temp_node.py file represent the relative position relationship between the robot and the pedestrians? How does the code model the interaction between people and between people and the robot? Is it done through distance?

**Comment by Shuijing123** [comment]
Yes, the interaction is weighted by [robot-human (RH) attention network](https://github.com/Shuijing725/CrowdNav_Prediction_AttnGraph/blob/632b087c671e8b300a49a6668c3135bc3a551489/rl/networks/selfAttn_srnn_temp_node.py#L318).
